### ISSUE TITLE
Support for PyTorch 1.1 Tensorboard support

### ISSUE THREAD

**Issue by activatedgeek** [wandb/client#327]
* Weights and Biases version: 0.7
* Python version: 3.7
* Operating System: CentOS Linux release 7.6.1810 (Core)

### Description

I was trying to use the new [Tensorboard support in PyTorch 1.1](https://docs.wandb.com/docs/integrations/tensorboard.html) with [Tensorboard 1.14 pre-release](https://pypi.org/project/tb-nightly/).

I understand that this Tensorboard/Tensorflow 1.14 is unreleased yet but should be out soon so hoping this issue gets worked on.

### What I Did

```python
trial = { ... }  # Trial configuration dict.

wandb.init(project=trial.get('project'),
           name=trial.get('name'),
           dir=trial.get('log_dir'),
           config=trial,
           tensorboard=True)
```

Relevant traceback
```shell
  File "/miniconda3/envs/try/lib/python3.7/site-packages/wandb/__init__.py", line 608, in init
    util.get_module("wandb.tensorboard").patch()
```

**Comment by vanpelt** [comment]
Hey @activatedgeek we've been working on getting this supported.  You can try what we have so far by installing wandb with the following:

`pip install --upgrade git+git://github.com/wandb/client.git@feature/tensorflow2#egg=wandb`

Let me know if that doesn't work.  We're hoping to cut a new release in the next few days.

**Comment by activatedgeek** [comment]
@vanpelt Awesome! I can confirm this works on this feature branch. Keeping this issue open until merged.

**Comment by tshrjn** [comment]
This feature branch doesn't work for me. I also tried with a simple demo code I created [here](https://gist.github.com/tshrjn/7513bd26efaa20eac4174b7a874207f9).

Using `tb-nightly==1.14.0a20190512`, `torch==1.1.0` and wandb from the feature branch.

**Comment by vanpelt** [comment]
Hey @tshrjn it looks like this one worked: https://app.wandb.ai/tshrjn/mnist_demo/runs/cpbu4pk1?workspace=user-vanpelt  I'm aware of some funkyness with pytorch and tensorboard I'll be debugging more before cutting the next release.

**Comment by tshrjn** [comment]
Oh sorry, the later runs are with tensboardX SummaryWriter and not the one from `torch.utils.tensorboard import SummaryWriter`. I'll rename and make it clear to understand.

**Comment by vanpelt** [comment]
Hey @tshrjn I just pushed a fix for this to the tensorflow2 branch.  We're hoping to get a release out ASAP.  You can try it now with: `pip install --upgrade git+git://github.com/wandb/client.git@feature/tensorflow2#egg=wandb`

**Comment by tshrjn** [comment]
Hi, @vanpelt It works for me now, thanks!  Though I am not sure of other edge cases, perhaps you can use some test-cases and then close the issue if they pass.

**Comment by vanpelt** [comment]
Fixed in 0.8.0
